napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -221,9 +237,20 @@ def _set_table(self, layer_str: str = ''):
             # Go through all the actions in the layer and add them to the table.
             for row, (action_name, action) in enumerate(actions.items()):
 
-                shortcuts = action_manager._shortcuts.get(action_name, [])
+                if isinstance(action, Action):  # action is registered through keymap handler
```

Comment:
This is a little confusing as `Action` is also a class in app-model. Maybe import as something else or import `action_manager` module and use `action_manager.Action`?